conda_requirements.txt

```diff
@@ -11,3 +11,4 @@ service_identity
 # cython is a lame requirement from rabbit hole of cartopy
 cython
 cartopy
+mastodon.py
```

Comment:
Couldn't tell what repo you're using, but this is what it's called in `conda-forge`.

Reply:
FYI, the magic there happens in my [ci_tooling repo](https://github.com/akrherz/ci_tooling/blob/main/miniconda.sh).  I am very impressed that you figured out as much as you did with all my clunker stuff!

Reply:
Oh neat, we should probably point our science users at `mamba` given the latest `conda` ToS.